### ISSUE TITLE
Config: low contrast

### ISSUE THREAD

**Issue by reynir** [robur-coop/dnsvizor#88]
> The contrast between the small text and background is not sufficient. The contrast ratio is 3.61:1 when it should be 4.5:1 for text smaller than 18px.

<img width="235" height="57" alt="Image" src="https://github.com/user-attachments/assets/f0cd8150-bf1f-406f-aaa8-355bb1d267e8" />

[1.4.3 Contrast (minimum)](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum)
